### DISCUSSION TITLE
Separate normalization

### DISCUSSION THREAD

**Discussion by GOzorun** [vdemichev/DiaNN#1212] [Q&A]
Hi,

I want to normalize extracellular matrix proteins separately and intracellular proteins separately to prevent more abundant intracellular proteins potentially masking the changes in ECM proteins. Does this mean I need to turn off normalization when running DIA-NN to process samples and go from non-normalized data, or do I still keep default normalization on and use pg_matrix file to separately normalize ECM and intracellular proteins? 

Thanks in advance!

**Comment by vdemichev** [comment]
Hi,

I would recommend to rather re-normalise the normalised quantities produced by DIA-NN.

Best,
Vadim

**Reply by GOzorun** [reply]
Thank you for the quick response!
